### DISCUSSION TITLE
Error: statfs examples/compose/local/traefik/certs: no such file or directory

### DISCUSSION THREAD

**Discussion by jay-elizaga-dev** [authelia/authelia#8467] [Documentation Feedback] [answered]
### Version

v4.38.17

### Deployment Method

Docker

### Reverse Proxy

Traefik

### Description

Encountering the error

```Error: statfs examples/compose/local/traefik/certs: no such file or directory``` 

when running `authelia/examples/compose/local/setup.sh`

on Docker/Podman.

### Reproduction

Follow the instructions provided here : https://www.authelia.com/integration/deployment/docker/#local

1. Run ./setup.sh.
2. The bug will show up when the traefik/cert directory is missing (and it will be missing unless it is manually created prior to running setup.sh)

### Build Information

```shell
This applies prior to Authelia being created so build-info not needed.  Can provide if requested.
```

### Logs (Authelia)

```shell
This applies prior to Authelia being created so logs not needed.  Can provide if requested.
```

### Generative AI

No

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] This is a bug report and not a support request

- [X] I have read the security policy and this bug report is not a security issue or security related issue

- [X] I have either included the complete configuration file or I am sure it's unrelated to the configuration

- [X] I have either included the complete debug / trace logs or the output of the build-info command if the logs are not relevant

- [X] I have provided all of the required information in full with the only alteration being reasonable sanitization in accordance with the [Troubleshooting Sanitization](https://www.authelia.com/r/sanitize) reference guide

- [X] I have checked for related proxy or application logs and included them if available

- [X] I have checked for related issues and checked the documentation

**Comment by jay-elizaga-dev** [comment]
I have the solution, currently creating the PR.

**Comment by james-d-elliott** [comment, marked as answer]
Issues are for reporting bugs or feature requests with the authelia binary/application. Not the documentation or documentation supporting resources. I'll convert this appropriately.

In addition this can't be replicated (docker creates all bind mounts as directories if the path doesn't exist):

```bash
# ls -aln traefik/
total 8
drwxr-xr-x 2 0 0 4096 Dec 16 23:57 .
drwxr-xr-x 3 0 0 4096 Dec 16 23:56 ..

# docker run -a stdout -v $PWD/traefik/certs:/tmp/certs authelia/authelia authelia crypto certificate rsa generate --common-name="*.${DOMAIN}" --directory=/tmp/certs/ > /dev/null

# ls -aln traefik/
total 12
drwxr-xr-x 3 0 0 4096 Dec 16 23:57 .
drwxr-xr-x 3 0 0 4096 Dec 16 23:56 ..
drwxr-xr-x 2 0 0 4096 Dec 16 23:57 certs

# ls -aln traefik/certs/
total 16
drwxr-xr-x 2 0 0 4096 Dec 16 23:57 .
drwxr-xr-x 3 0 0 4096 Dec 16 23:57 ..
-rw------- 1 0 0 1704 Dec 16 23:57 private.pem
-rw------- 1 0 0 1103 Dec 16 23:57 public.crt
```

**Reply by jay-elizaga-dev** [reply]
Thank you for putting it into the appropriate space.  You are right. In docker documentation, it   Maybe this is more of an issue with Podman or some specific build of docker.  

https://github.com/containers/podman/issues/6234 

Seems there's some controversy whether this should be the case or not. 

Thank you for your time, James, it was a pleasure.
